packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -609,6 +613,17 @@ export default class TabDisks extends Component {
       )
   )
 
+  _rescanIsoSrs = async () => {
+    this.setState({ isScanningIsoSr: true })
+    const isoSrs = filter(this.props.srs, ({ SR_type }) => SR_type === 'iso')
```

Comment:
I don't think you'll get all the ISO SRs that need to be rescanned like that, since `props.srs` are the VM's VDIs' SRs and the VDIs themselves are filtered to only get the non-CD ones: https://github.com/vatesfr/xen-orchestra/blob/3c352a3545cfd5c33b1199b76b5b35a2fb2b78a0/packages/xo-web/src/common/selectors.js#L507